config/samples/example-test.sh

```diff
@@ -0,0 +1,193 @@
+	kind export logs --name=${KindName} $LOG_DIR
+	sudo chown -R $USER:$USER $LOG_DIR
+
+	warning "debugInfo finished ! "
+	warning "This means that some tests have failed. Please check the log. 🌚"
+	debug=1
+	exit 1
+}
+trap 'debugInfo $LINENO' ERR
+trap 'debugInfo $LINENO' EXIT
+debug=0
+
+function cecho() {
+	declare -A colors
+	colors=(
+		['black']='\E[0;47m'
+		['red']='\E[0;31m'
+		['green']='\E[0;32m'
+		['yellow']='\E[0;33m'
+		['blue']='\E[0;34m'
+		['magenta']='\E[0;35m'
+		['cyan']='\E[0;36m'
+		['white']='\E[0;37m'
+	)
+	local defaultMSG="No message passed."
+	local defaultColor="black"
+	local defaultNewLine=true
+	while [[ $# -gt 1 ]]; do
+		key="$1"
+		case $key in
+		-c | --color)
+			color="$2"
+			shift
+			;;
+		-n | --noline)
+			newLine=false
+			;;
+		*)
+			# unknown option
+			;;
+		esac
+		shift
+	done
+	message=${1:-$defaultMSG}     # Defaults to default message.
+	color=${color:-$defaultColor} # Defaults to default color, if not specified.
+	newLine=${newLine:-$defaultNewLine}
+	echo -en "${colors[$color]}"
+	echo -en "$message"
+	if [ "$newLine" = true ]; then
+		echo
+	fi
+	tput sgr0 #  Reset text attributes to normal without clearing screen.
+	return
+}
+
+function warning() {
+	cecho -c 'yellow' "$@"
+}
+
+function error() {
+	cecho -c 'red' "$@"
+}
+
+function info() {
+	cecho -c 'blue' "$@"
+}
+
+function waitPodReady() {
+	namespace=$1
+	podLabel=$2
+	START_TIME=$(date +%s)
+	while true; do
+		readStatus=$(kubectl -n${namespace} get po -l ${podLabel} --ignore-not-found=true -o json | jq -r '.items[0].status.conditions[] | select(."type"=="Ready") | .status')
+		if [[ $readStatus == "True" ]]; then
+			info "Pod:${podLabel} ready"
+			kubectl -n${namespace} get po -l ${podLabel}
+			break
+		fi
+
+		CURRENT_TIME=$(date +%s)
+		ELAPSED_TIME=$((CURRENT_TIME - START_TIME))
+		if [ $ELAPSED_TIME -gt $TimeoutSeconds ]; then
+			error "Timeout reached"
+			kubectl describe po -n${namespace} -l ${podLabel}
+			kubectl get po -n${namespace} --show-labels
+			exit 1
+		fi
+		sleep 5
+	done
+}
+
+function waitCRDStatusReady() {
+	source=$1
+	namespace=$2
+	name=$3
+	START_TIME=$(date +%s)
+	while true; do
+		readStatus=$(kubectl -n${namespace} get ${source} ${name} --ignore-not-found=true -o json | jq -r '.status.conditions[0].status')
+		message=$(kubectl -n${namespace} get ${source} ${name} --ignore-not-found=true -o json | jq -r '.status.conditions[0].message')
+		if [[ $readStatus == "True" ]]; then
+			info $message
+			break
+		fi
+
+		CURRENT_TIME=$(date +%s)
+		ELAPSED_TIME=$((CURRENT_TIME - START_TIME))
+		if [ $ELAPSED_TIME -gt $TimeoutSeconds ]; then
+			error "Timeout reached"
+			exit 1
+		fi
+		sleep 5
+	done
+}
+info "1. create kind cluster"
+make kind
+
+info "2. install kubebb core"
+info "2.1 deploy kubebb/core and minio"
+helm repo add kubebb https://kubebb.github.io/components/
+helm repo update
+helm install -n arcadia --create-namespace kubebb-core kubebb/kubebb-core
```

Comment:
Seems you are trying to install kubebb which should be in namespace kubebb-system